tests/foreman/ui/test_puppetenvironment.py

```diff
@@ -53,21 +53,23 @@ def test_positive_end_to_end(session, module_org, module_loc):
             'locations.resources.assigned': [module_loc.name],
             'organizations.resources.assigned': [module_org.name],
         })
-        assert session.puppetenvironment.search(name)[0]['Name'] == name
+        found_envs = session.puppetenvironment.search(name)
+        assert name in [env['Name'] for env in found_envs]
         env_values = session.puppetenvironment.read(name)
         assert env_values['environment']['name'] == name
         assert env_values['organizations']['resources']['assigned'][0] == module_org.name
         assert env_values['locations']['resources']['assigned'][0] == module_loc.name
         session.puppetenvironment.update(name, {'environment.name': new_name})
-        assert session.puppetenvironment.search(new_name)[0]['Name'] == new_name
+        found_envs = session.puppetenvironment.search(new_name)
+        assert new_name in [env['Name'] for env in found_envs]
         session.puppetenvironment.delete(new_name)
         assert not session.puppetenvironment.search(new_name)
 
 
 @tier2
-def test_positive_availability_for_host_in_multiple_orgs(session, module_loc):
+def test_positive_availability_for_host_and_hostgroup_in_multiple_orgs(session, module_loc):
     """New environment that present in different organizations should be
-    visible for any created host in these organizations
+    visible for any created host nand hostgroup in these organizations
```

Comment:
`nand` typo

Reply:
s/New environment that present/A new environment that is present/ {why does it have to be new? You could just say "An environment that is present"}
s/in these organizations/in those organizations/